src/CMake/FindANARI.cmake

```diff
@@ -63,7 +65,8 @@ if(EXISTS ${VISIT_ANARI_DIR})
 endif()
 
 if(anari_FOUND)
-    add_definitions(-DVISIT_ANARI)
+    set(HAVE_LIBANARI true)
```

Comment:
Would it make any sense for these two symbols (the CMake variable `HAVE_LIBANARI` and the CPP define, `HAVE_ANARI`) to be the same?

Reply:
I can make the change. I was copying OSPRay and thought this was discussed when the PR was submitted for this work. Didn't want to re-litigate whatever the VisIt team decided on with Allen. 
```
set(HAVE_LIBOSPRAY true)
add_definitions(-DHAVE_OSPRAY)
```

Reply:
Ok, I see. Following OSPRay pattern certainly makes sense to me if that pattern is fully refined at this point. I honestly have no strong opinion. Maybe others do though.